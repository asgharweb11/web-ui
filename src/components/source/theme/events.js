import theme from "./theme"

export const Color = (item , param) => {
    switch(item){
        case "ligth" :
            return theme.ligth.text.grey;
        case "dark" :
            return theme.dark.text.grey;
    }
}