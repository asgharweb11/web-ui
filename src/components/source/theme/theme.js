export default {
    ligth : {
        main : "white",
        content : "repeating-linear-gradient(0deg , #f2efff , #f3f3ff)",
        primary : "#2196f3",
        secondary : "#ff80ab",
        error : "#f44336",
        warning : "#ff9800",
        info : "#2196f3",
        success : "#4caf50",
        text : {
            grey : "#313131",
            grey1 : "#656565",
            grey2 : "#909090",
            grey3 : "#808080",
            grey4 : "#9a9a9a",
            grey5 : "#c7c7c7",
            color : "#ffffff",
        },
        glass : {
            grey : "#000000a3",
            grey2 : "#00000066",
            grey3 : "#0000003b",
            grey4 : "#0000001c",
            grey5 : "#0000000d",
        },
        hover : {
            primary : "#246fab",
            secondary : "#ec5286",
            error : "#dc1709",
            warning : "#e08703",
            info : "#0d7ad2",
            success : "#237b26",
        }
    },  
    dark : {
        main : "#212121",
        content : "repeating-linear-gradient(0deg ,#333333 ,#333333)",
        primary : "#4dabf5",
        secondary : "#ff99bb",
        error : "#e57373",
        warning : "#ffb74d",
        info : "#64b5f6",
        success : "#81c784",
        text : {
            grey : "#ffffff",
            grey1 : "#ececec",
            grey2 : "#c7c7c7",
            grey3 : "#212121",
            grey4 : "#6b6b6b",
            grey5 : "#464646",
            color : "2b2b2b"
        },
        glass : {
            grey : "#ffffffb3",
            grey2 : "#ffffff87",
            grey3 : "#ffffff59",
            grey4 : "#ffffff2b",
            grey5 : "#ffffff0f",
        },
        hover : {
            primary : "#90ceff",
            secondary : "#ffb5ce",
            error : "#fdaaaa",
            warning : "#ffd393",
            info : "#a8d8ff",
            success : "#b0efb2",
        }

    },
    fontSize : {
        h1 : "40px",
        xl : "25px",
        lg : "20px",
        md : "16px",
        sm : "13px",
        xs : "10px",
    },
    px : {
        p1 : "0px",
        p2 : "1px",
        p3 : "5px",
        p4 : "10px",
        p5 : "15px",
        p6 : "20px",
    },
    media : {
        mobileS: '320px',
        mobileM: '375px',
        mobileL: '425px',
        tablet: '768px',
        tabletL : "991px",
        laptop: '1024px',
        laptopM: '1200px',
        laptopL: '1440px',
        desktop: '2560px'
    }

}