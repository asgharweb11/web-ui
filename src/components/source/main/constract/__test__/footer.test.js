import React from "react"
import renderer from "react-test-renderer"
// -------- Context
import Context from "../../../../../context.js"
// -------- Styled-ThemeProvider
import {ThemeProvider} from "styled-components"
import theme from "../../../theme/theme"
//--------- Component
import Footer from "../footer"


test('Component : Constract/footer.js', () => {
    const container = renderer.create(<Context>
        <ThemeProvider theme={theme}>
            <Footer />
        </ThemeProvider>
    </Context>)

    let tree = container.toJSON();
    expect(tree).toMatchSnapshot()
})
