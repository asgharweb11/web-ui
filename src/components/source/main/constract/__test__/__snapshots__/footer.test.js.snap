// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component : Constract/footer.js 1`] = `
<div
  className="sc-bdfBwQ ajUNk"
>
  <div
    className="card-Profile"
  >
    <div
      className="user"
    >
      <img
        alt="profile user"
        src="./files/pic/user/1.png"
      />
      <h3>
        Jessica
      </h3>
      <p>
        supplies
      </p>
    </div>
    <div
      className="user"
    >
      <img
        alt="profile user"
        src="./files/pic/user/2.png"
      />
      <h3>
        Martha
      </h3>
      <p>
        Cook
      </p>
    </div>
    <div
      className="user"
    >
      <img
        alt="profile user"
        src="./files/pic/user/3.png"
      />
      <h3>
        Sofia
      </h3>
      <p>
        Driver
      </p>
    </div>
    <div
      className="user"
    >
      <img
        alt="profile user"
        src="./files/pic/user/4.png"
      />
      <h3>
        jan
      </h3>
      <p>
        marjting
      </p>
    </div>
    <div
      className="user"
    >
      <img
        alt="profile user"
        src="./files/pic/user/5.jpg"
      />
      <h3>
        Jessica
      </h3>
      <p>
        Cook
      </p>
    </div>
    <div
      className="user"
    >
      <img
        alt="profile user"
        src="./files/pic/user/6.jpg"
      />
      <h3>
        Mark
      </h3>
      <p>
        marjting
      </p>
    </div>
    <div
      className="user"
    >
      <img
        alt="profile user"
        src="./files/pic/user/7.jpg"
      />
      <h3>
        jack
      </h3>
      <p>
        Cook
      </p>
    </div>
  </div>
  <div
    className="addUser"
  >
    <svg
      aria-hidden="true"
      className="svg-inline--fa fa-plus fa-w-14 "
      data-icon="plus"
      data-prefix="fas"
      focusable="false"
      role="img"
      style={Object {}}
      viewBox="0 0 448 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M416 208H272V64c0-17.67-14.33-32-32-32h-32c-17.67 0-32 14.33-32 32v144H32c-17.67 0-32 14.33-32 32v32c0 17.67 14.33 32 32 32h144v144c0 17.67 14.33 32 32 32h32c17.67 0 32-14.33 32-32V304h144c17.67 0 32-14.33 32-32v-32c0-17.67-14.33-32-32-32z"
        fill="currentColor"
        style={Object {}}
      />
    </svg>
  </div>
</div>
`;
